excalidraw/excalidraw
# Fix/Feature: Close Dialog "Save To" on Save To Disk

**Description:**

This pull request addresses a user flow issue by adding functionality to close the dialog box when the "Save to File" action is performed. 

**Issue:**

More information about the issue can be found on the following GitHub issue created by [@starlove54](https://github.com/starlove54): [excalidraw/excalidraw#7156](https://github.com/excalidraw/excalidraw/issues/7156)

**Changes Made:**

I have made this PR under the issue mentioned above. Before this change, when a user clicked the "Save to File" button, the dialog box for "Save to" was not disappearing, resulting in a suboptimal user experience. This pull request resolves this issue by adding the functionality to close the dialog box upon clicking the "Save to File" button.

**How to Test:**

To test this change, follow these steps:
1. Open Excalidraw.
2. Create or open a drawing.
3. Click the 'Save To' button on the side menu.
4. Click the "Save to File" button.
5. The "Save to" dialog box should now close after clicking "Save to File."

**Checklist:**

- [x] I have tested this change thoroughly.

I look forward to your feedback and any necessary adjustments to ensure the seamless integration of this feature into Excalidraw.

Thank you!
